src/PaginationPage.js

```diff
@@ -6,14 +6,18 @@ export default class PokePage extends React.Component {
     state = {
```

Comment:
I commented on Canvas, but you didn't necessarily have to create a new page.  This doesn't matter in the end, but just know in the future, you may want to incorporate pagination in your results you already had.